src/mds/Server.h

```diff
@@ -347,6 +347,8 @@ class Server {
   void handle_conf_change(const ConfigProxy& conf,
                           const std::set <std::string> &changed);
 
+ // int worm_state_transition(Inode *in, int op);
+
```

Comment:
This doesn't look related to the rest of the patch.

Reply:
already deleted